packages/framework/src/client/resource-request.js

```diff
@@ -0,0 +1,111 @@
+import debugFactory from 'debug';
+import { isDateEarlier } from '../utils/dates';
+import { SECOND } from '../utils/constants';
+
+const DEFAULT_TIMEOUT = 30 * SECOND;
```

Comment:
There's already a default defined for `timeout`, it's 20 seconds: https://github.com/Automattic/fresh-data/blob/69bebbeed661d914a788712c2a4807348e31efd6/packages/framework/src/client/requirements.js#L5

Will this replace the other constant?

Reply:
Yes, it will. Scheduler and ResourceRequest will both be replacing much of the code currently in `index.js`/APIClient.